recipe/build_kaldi.sh

```diff
@@ -77,13 +77,24 @@ mkdir build
 
 pushd build
```

Comment:
Let's not duplicate the invocations like that. This can be done in a much more targeted manner:

```suggestion
pushd build

if [[ "${target_platform}" == "linux-64" ]] && [[ "${cuda_compiler_version}" =~ 12.* ]]; then
    # maybe add a short comment about why
    CMAKE_ARGS="${CMAKE_ARGS} -DCUDA_TOOLKIT_ROOT_DIR=${PREFIX}/targets/x86_64-linux"
fi

cmake ${CMAKE_ARGS} \
    ...
```